api/capi/src/nnstreamer-capi-single-new.c

```diff
@@ -176,6 +176,9 @@ invoke_thread (void *arg)
       for (i = 0; i < single_h->out_info.num_tensors; i++) {
         out_data->tensors[i].tensor = out_tensors[i].data;
       }
+    } else {
+      for (i = 0; i < single_h->out_info.num_tensors; i++)
+        g_free (out_tensors[i].data);
```

Comment:
Please describe (comment) what the situation is when this ```else``` statement is used.

Reply:
updated.

Reply:
Good! I was wondering how it could be FALSE :)

Reply:
Earlier, `data_ready` was set to false in case of timeout by the caller. However, there was still a case missing when memory of output wasn't freed. Thanks for helping in pointing that out.